config/routes.rb

```diff
@@ -102,7 +103,7 @@
     get '/revision_history/:id/revert_object' => 'versions#revert_object', as: 'revision_history_revert_object'
     get '/revision_history/:id/revert_attribute' => 'versions#revert_attribute', as: 'revision_history_revert_attribute'
   end
-
+  resources :organizations
```

Comment:
Since we only have the `index` action, it's better to say:

`resources :organizations, only: [:index]`